tests/sql/helpers.test.js

```diff
@@ -0,0 +1,59 @@
+const { sqlToCamelCased, aggregateIntoJson } = require('../../src/sql/helpers');
+
+describe('sqlToCamelCased', () => {
```

Comment:
what about a test for converting from camel case to snake case?

Reply:
depends on how [this](https://github.com/hms-dbmi-cellenics/api/pull/324#discussion_r832999964) is resolved